cmd/testscript/testdata/nogo.txt

```diff
@@ -1,7 +1,7 @@
 # should support skip
 unquote file.txt
 
-env PATH=
+dropgofrompath
```

Comment:
How about a comment here?

```
# We can't just set PATH to empty
# because we need the part of it that
# contains the executable names, so use
# a special builtin instead.
```
?